Wireframe/css/style.css

```diff
@@ -0,0 +1,141 @@
+:root {
+  --font-family: sans-serif;
+  --orange-dark: #ef6419;
+  --aqua-color: aqua;
+  --grey-color: grey;
+}
+
+html,
+body {
+  box-sizing: border-box;
+  background-color: #ffeead;
+  height: 100%;
+  padding: 10px;
+  margin: 0px;
+}
+
+.container {
+  height: 100%;
+  display: grid;
+  grid-gap: 6px;
+  /* grid-template-columns: [main-start] 1fr [content-start] 5fr [content-end main-end];
+  grid-template-rows: [main-start] 40px [content-start] auto [content-end] 40px [main-end]; */
+  grid-template-columns: repeat(12, 1fr);
+  grid-template-rows: 40px 510px 50px 10px;
+  grid-template-areas:
+    "h h h h h h h h h h h h"
+    "c c c c c c c c c c m m"
+    "c c c c c c c c c c m m"
+    "l l l l l l r r r r m m"
+    "l l l l l l r r r r m m"
+    "f f f f f f f f f f f f";
+}
+
+.header {
+  grid-area: h;
+  /* grid-column: main; */
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #e0e1dd;
+  border-radius: 8px;
+  position: float;
```

Comment:
What is this line doing? Inspect this line in Devtools and think about what it is for.